demo/stream_audio_out/run.py

```diff
@@ -0,0 +1,40 @@
+import gradio as gr
+import numpy as np
+from pydub import AudioSegment
+import time
+
+
+def stream_audio(audio_file, lag):
+    audio = AudioSegment.from_mp3(audio_file)
+    chunk_length = 1000
+    chunks = []
+    while len(audio) > chunk_length:
+        chunks.append(audio[:chunk_length])
+        audio = audio[chunk_length:]
+    if len(audio):  # Ensure we don't end up with an empty chunk
+        chunks.append(audio)
+
+    for i, chunk in enumerate(chunks):
+        file = f"/tmp/{i}s.mp3"
+        chunk.export(file, format="mp3")
+        time.sleep(lag)
+        yield file, i
```

Comment:
So you create the list of chunks and then yield then one at a time. To simplify the demo (and maybe make it more realistic), why not just yield each chunk directly as you get it?